### PR TITLE
fix RUN-1067 - `isInstanceOfNative`, small object preview bugs

### PR DESCRIPTION
**Pull request by Domiii**
* fix `isInstanceOfNative` (its still a hackfix - better solution pending)
* update `makeDebuggeeValue` to use the more resilient `convertCborToJS` utility (has more error reporting)

### CODE REVIEW

File: third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc [hunk 1]
```diff
@@ -535,6 +535,8 @@ function isInstanceOfNative(x, target) {
   const name = target?.name;
   return name &&
     x?.constructor?.toString()?.includes('() { [native code] }') &&
+    // fix https://linear.app/replay/issue/RUN-1067/chromium-css-style-object-preview-errors-out
```
Comment (bhackett1024) [inline third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc:538]: This comment isn't necessary, git blame can be used to recover this information and comments should generally be immediately descriptive / informative.

File: third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc [hunk 2]
```diff
@@ -3210,7 +3212,9 @@ static void fromJsMakeDebuggeeValue(
     const v8::FunctionCallbackInfo<v8::Value>& args) {
   v8::Isolate* isolate = args.GetIsolate();
 
-  CHECK(args.Length() == 1 && "must be called with a single value");
+  // CHECK(args.Length() == 1 && "must be called with a single value");
```
Comment (bhackett1024) [inline third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc:3215]: rm commented out check

**Review by bhackett1024 — APPROVED** [review]

File: third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc (hunk 1 above)
Comment (Domiii) [reply third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc:538]: Ok

File: third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc (hunk 2 above)
Comment (Domiii) [reply third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc:3215]: Bad habit slipped through :eyes:

**Pull request merged**